### ISSUE TITLE
请教下， 为什么KLJSLoss返回多不是一个值，而是一组batchsize长度的值

### ISSUE THREAD

**Issue by sofiawu** [PaddlePaddle/PaddleOCR#4087]
文件检测CML训练时，发现KLJSLoss计算时特意指定了axis
loss = paddle.mean(loss, axis=[1, 2])
这样计算得到的loss不是一个值。
为什么不直接 paddle.mean(loss）呢

**Comment by littletomatodonkey** [comment]
你好，mean在后面也会做的，这里保留维度信息，可以做进一步的计算

**Comment by sofiawu** [comment]
请问做了什么进一步的计算？看代码感觉直接就backward了
